components/pubsub.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: pubsub
+spec:
+  type: pubsub.redis
+  metadata:
+  - name: redisHost
+    value: localhost:6379
+  - name: redisPassword
+    value: ""
```

Comment:
What are these components used for? If these are for tests, they should be moved inside `test/components`.

Reply:
My bad, I included them by mistake - Removed now.